homeassistant/components/stream/core.py

```diff
@@ -110,41 +120,71 @@ def idle(self) -> bool:
         return self._idle_timer.idle
 
     @property
-    def segments(self) -> list[int]:
+    def sequences(self) -> list[int]:
```

Comment:
Just a renaming that should be more descriptive, as segments could mean data or numbers